src/applications/disability-benefits/all-claims/tests/schema/maximal-test.json

```diff
@@ -107,7 +107,7 @@
       }
     },
     "view:unemployabilityUploadChoice": "answerQuestions",
-    "view:unemployable": true,
+    "view:unemployable": false,
```

Comment:
This took us down a path that we had no data for. After we get this tester fully-functional, we should grab a whole new set of data for this file; it no longer lives up to its name.